hub/core/compute/ray.py

```diff
@@ -5,6 +5,7 @@
 
 class RayProvider(ComputeProvider):
     def __init__(self, workers):
+        super().__init__(workers)
         if not ray.is_initialized():
             ray.init()
         self.workers = workers
```

Comment:
we can get rid of this line then if we're calling the base class' init

Reply:
Done